### DISCUSSION TITLE
Can you add gold adjustment

### DISCUSSION THREAD

**Discussion by mistrornge** [denbkh/stoneshard-save-editor#5] [Ideas]
It’s called stack I believe in game.  Thank you

**Comment by denbkh** [comment]
Hi, right now I have no plan for implementing inventory editing.

**Reply by mistrornge** [reply]
One of the reasons I suggested it was leveling your character or even respec is going to be costly in gear.  You will be starting at a disadvantage until they can gear up to fight the now leveled mobs.
